dev/build/windows/makecoq_mingw.sh

```diff
@@ -383,7 +383,7 @@ function build_conf_make_inst {
 function install_glob {
   # Check if any files matching the pattern exist
   if [ "$(echo $1)" != "$1" ] ; then
-    install -D -t $2 $1
+    install -D -t "$2" "$1"
```

Comment:
This is incorrect and causes a failure as `$1` is a glob.

Reply:
Also it may be better to rewrite this so there's less messing with globs.
For instance 
```bash
function install_glob {
    if [ -e "$2" ]; then
        install -D -t "$@"
    fi
}
```
This involves a change in calling convention:
```bash
install_glob "$target" "$foo"*"bar" # note glob not quoted and argument order changed
```

Reply:
I agree that it would be more elegant to switch the arguments and do the glob expansion in the call to install_glob.
Also if someone has an elegant way to handle null globs in bash - please educate me. Preferably the test inside install_glob should be on the argument count then.

Reply:
> Also if someone has an elegant way to handle null globs in bash - please educate me. Preferably the test inside install_glob should be on the argument count then.

Do we set the bash option for expanding null globs to an empty array? If yes then we can test argument count, otherwise it expands to itself (ie `*` => `*`) and we have to test whether that's a valid file.

Reply:
nullglob is not set in my scripts - a lot of commands do unintended things if this is set , e.g. a grep bla *.txt would read from stdin instead of failing or returning nothing. And afaik bash has no way to do it just in one case.

Reply:
Maybe we should use find instead of glob - it is a bit more to write in the calls to install_glob, but I think it is a more maintainable solution.

Reply:
I think globs are fine for this.

Reply:
Fixed.